portal/static/js/main.js

```diff
@@ -1255,6 +1255,93 @@ var fillContent = {
                 };
             });
         };
+    },
+    "emailContent": function(userId, messageId) {
```

Comment:
Would it make sense to move these functions to a separate file? Most of this functionality will only be on the /communicate page right?

Reply:
hmm, this is also used on the profile page as well.  so refactoring will need to take that into consideration.